### PR TITLE
Added damerau-levenshtein types

### PR DESCRIPTION
**Pull request by g3ru1a**
Please fill in this template.

- [ x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x ] Test the change in your own code. (Compile and run.)
- [ x ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ x ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ x ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ x ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ x ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ x ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [ x ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@g3ru1a thx!

**Review by tanelso2 — APPROVED** [review]

File: types/damerau-levenshtein/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Type definitions for damerau-levenshtein 1.0
+// Project: https://github.com/tad-lispy/node-damerau-levenshtein#readme
+// Definitions by: G3ru1a <https://github.com/me>
```
Comment (peterblazejewicz) [inline types/damerau-levenshtein/index.d.ts:3]:
```suggestion
// Definitions by: G3ru1a <https://github.com/g3ru1a>
```

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
@g3ru1a your github's handle needs to be corrected thx!

**Comment by g3ru1a** [comment]
Updated it, thanks @peterblazejewicz !

**Review by peterblazejewicz — APPROVED** [review]

**Pull request merged**